CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## [next]
+ - Improve error handling for the onLoad functiomaya jama dioramabig fatn
```

Comment:
Either you just went "keyboard smash" or this is should be translated, either way I think it should be changed

Reply:
hahahaha that was an accidental control v from my google search :p

Reply:
do recommend the big fat quiz of the year, 2020 edition just got out but they are all good

Reply:
Ah in that case, let me **not** pass that on my Google search haha